tests/roles/backend_services/tasks/main.yaml

```diff
@@ -1,3 +1,54 @@
+- name: check if the source SC exists and can be used for the local-storage SC
+  no_log: "{{ use_no_log }}"
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc get sc {{ storage_class_name }} -o yaml
+  register: local_sc_result
+  failed_when:
+    - local_sc_result.rc != 0
+    - ('Error from server (NotFound)' not in local_sc_result.stdout)
+  changed_when: local_sc_result.rc != 0
+
+- name: create a local-storage SC from the given source SC
+  when: not local_sc_result.changed # noqa: no-handler
+  no_log: "{{ use_no_log }}"
+  vars:
+    _sc_params:
+      reclaimPolicy: "{{ storage_reclaim_policy }}"
+      metadata:
+        resourceVersion: ""
+        uuid: ""
+        creationTimestamp: ""
+        annotations: ""
+        name: local-storage
+  ansible.builtin.shell: |
+    {{ shell_header }}
+    {{ oc_header }}
+    oc delete --ignore-not-found=true sc local-storage
+    echo -e '{{ local_sc_result.stdout | from_yaml | combine(_sc_params) | to_nice_yaml }}' | oc apply -f -
```

Comment:
Maybe i don't fully grasp how this creation works but it's weird to me that there is no mention of the `storage_class_name` variable here at all? How does it refer to the "given source SC" then?

Other than that i'm ok with the change. I was fine with doing the crc storage cleanup/recreate manually but if this approach makes the storage recreation work "out of the box" then +1.

Reply:
Was just confused with `changed_when: local_sc_result.rc != 0` which i parsed wrongly as if it was `  changed_when: local_sc_result.rc == 0`. I understand now :).

Reply:
as explained on slack:
It copied SC from the source SC,
or creates one based on a default CRC SC, if no source to copy from.
To override this 'storage_class_name: lvms-magic-sc', that lvms will become a new source for local-storage SC
The next iteration will be to support direct renaming SC in all places